exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/ParquetRecordReader.java

```diff
@@ -105,13 +107,15 @@
   long totalRecordsRead;
   private final FragmentContext fragmentContext;
 
+  public ParquetReaderStats parquetReaderStats = new ParquetReaderStats();
```

Comment:
I did have this as final originally, but close is called multiple times so I had to remove the final.